### ISSUE TITLE
Zeichenkette mit Komma wird falsch dargestellt

### ISSUE THREAD

**Issue by Sascha8** [ioBroker/ioBroker.mqtt#460]
Hi,

schicke ich über Mosquitto den String `1,2,3` wird die Ausgabe mit nicht druckbaren Zeichen dargestellt.

Debug-Log für String `Test123`, `Test,1,2,3` und `1,2,3` Test123 und Test1,2,3 laufen fehlerfrei.

```
stateChange mqtt.0.dev.test: {"val":"Test123","ack":true,"ts":1719766597354,"q":0,"from":"system.adapter.mqtt.0","user":"system.user.admin","lc":1719766597354}

stateChange mqtt.0.dev.test: {"val":"Test,1,2,3","ack":true,"ts":1719766603869,"q":0,"from":"system.adapter.mqtt.0","user":"system.user.admin","lc":1719766603869}

stateChange mqtt.0.dev.test: {"val":"\u0001\u0002\u0003","ack":true,"ts":1719766607776,"q":0,"from":"system.adapter.mqtt.0","user":"system.user.admin","lc":1719766607776}
```

**Versions:**  
 - Adapter version: 5.2.0
 - JS-Controller version: 5.0.19
 - Node version: 18.19.1
 - Operating system: Linux Debian

![image](https://github.com/ioBroker/ioBroker.mqtt/assets/44703260/489a7b38-fe11-47c8-8fec-81880618b21e)

![image](https://github.com/ioBroker/ioBroker.mqtt/assets/44703260/fc5809aa-0e8f-4b62-b9a6-ad6e96c4c255)

**Comment by Apollon77** [comment]
Ich denke das liegt daran das diese kodierung üblicherweise genutzt wird um Binäre zeichen zu übertragen ... daher kommt es als binärzeichen an.

**Comment by Sascha8** [comment]
Ja, so siehts im Debug log aus. Jeder andere Client stellt es aber als String, wie es definiert ist, dar.

**Comment by Apollon77** [comment]
Ich glaube das Haupt Dilemma ist das MQTT Daten immer Context spezifisch sind und die Konvertierung die hier drin ist ist wohl drin weil einige Clients binäre daten als kommaseparierte liste an zahlen übertragen und bestimmt itrgendewann mal jemand gefordert hat das als Binäre daten zu verstehen.

Am Ende kann man das rausnehmen was dazu führt das es ein breaking change ist ... ich schaue mal
